NitroxPatcher/Patches/Dynamic/Planter_AddItem_Patch.cs

```diff
@@ -7,15 +7,21 @@ namespace NitroxPatcher.Patches.Dynamic;
 
 public sealed partial class Planter_AddItem_Patch : NitroxPatch, IDynamicPatch
 {
-    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((Planter p) => p.AddItem(default));
+    public static readonly MethodInfo TARGET_METHOD = Reflect.Method((Planter p) => p.AddItem(default, default));
 
-    public static void Prefix(InventoryItem item)
+    public static void Postfix(Plantable plantable, int slotID, Planter __instance)
```

Comment:
I think we usually put instance first to be consistent
```suggestion
    public static void Postfix(Planter __instance, Plantable plantable, int slotID)
```